### ISSUE TITLE
Simplified and Traditional Chinese Conversion

### ISSUE THREAD

**Issue by EnterMan123** [RSSNext/Folo#1259]
### Clear and concise description of the problem

🔻Applicable to body text and titles

https://github.com/user-attachments/assets/7e055576-10ba-4cd7-9f31-11562790fb16

### Suggested solution

X

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by EnterMan123** [comment]
无缝转换繁简体原文

**Comment by EnterMan123** [comment]
TEST 

测试

**Comment by hyoban** [comment]
I think this can be achieved through the translate feature

**Comment by EnterMan123** [comment]
翻译功能对此无效，您可以进行测试
rsshub://udn/news/breakingnews/5

**Comment by hyoban** [comment]
Indeed, this is because https://www.npmjs.com/package/franc-min can only recognize Mandarin Chinese
